src/components/UI/Overlay/OverlayContent/ProviderTimeoutMessage.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FunctionComponent } from "react";
+import { useOverlayContext } from "../../../../common/context/OverlayContext";
+import warningIcon from "../../../../../src/static/images/Alert/warning.png";
+import { Button } from "../../Button";
+
+interface ProviderTimeoutMessageProps {
+  address: string;
+}
+
+export const ProviderTimeoutMessage: FunctionComponent<ProviderTimeoutMessageProps> = ({address}) => {
+  const { closeOverlay } = useOverlayContext();
+
```

Comment:
looks quite similar to OverlayContent provider, can consider to reuse but will probably need to refactor [`isSuccess`](https://github.com/TradeTrust/tradetrust-ui-components/blob/master/src/components/UI/Overlay/OverlayContent/OverlayContent.tsx#L20) in deciding which icon to render. currently only tick (success) and cross (error), this new addition will include exclamation (warning)

Reply:
right i forgot we have a "confirmation look" that looks similar, thanks